doc/man3/SSL_CTX_set_verify.pod

```diff
@@ -107,12 +107,11 @@ application provided procedure also has access to the verify depth information
 and the verify_callback() function, but the way this information is used
 may be different.
 
-SSL_CTX_set_verify_depth() and SSL_set_verify_depth() set the limit up
-to which depth certificates in a chain are used during the verification
-procedure. If the certificate chain is longer than allowed, the certificates
-above the limit are ignored. Error messages are generated as if these
-certificates would not be present, most likely a
-X509_V_ERR_UNABLE_TO_GET_ISSUER_CERT_LOCALLY will be issued.
+SSL_CTX_set_verify_depth() and SSL_set_verify_depth() set the maximum acceptable
+chain depth to be considered during the verification procedure. Neither
+end-entity nor the trust-anchor certificates count against B<depth>. If the
+certificate chain needed to reach a trusted issuer is longer than B<depth+2>,
+X509_V_ERR_CERT_CHAIN_TOO_LONG will be issued.
```

Comment:
This no as clear as the documentation in verify(1), which explains the depth limit as maximum number of intermediate CA certificates *between* the leaf (end-entity or EE) certificate and the trust-anchor (TA).  A maximum length chain thus has the EE certificate at depth 0, intermediate certificates at depths `1, ..., limit`, and the TA at depth `limit + 1` for a total of `limit + 2` chain elements.